### PR TITLE
Document :dirs option

### PR DESCRIPTION
**Pull request by nathanl**

### CODE REVIEW

File: lib/phoenix_live_reload/live_reloader.ex [hunk 1]
```diff
@@ -66,6 +66,25 @@ defmodule Phoenix.LiveReloader do
...
+          |> Path.join("..")
+          |> Path.expand()
+
+        config :phoenix_live_reload, :dirs, [
+          Path.join([root_path, "apps", "app1"]),
+          Path.join([root_path, "apps", "app2"]),
+        ]
+
+    You'll also want to be sure that the configured `:patterns` for
+    `live_reload` will match files in the sister application.
```
Comment (josevalim) [inline lib/phoenix_live_reload/live_reloader.ex:87]: Can you please outdent everything by 2 spaces?

File: lib/phoenix_live_reload/live_reloader.ex [hunk 2]
```diff
@@ -66,6 +66,25 @@ defmodule Phoenix.LiveReloader do
       Useful when class names are determined at runtime, for example when
       working with CSS modules. Defaults to false.
 
+    In an umbrella app, if you want to enable live reloading based on code
+    changes in sister applications, set the `reloadable_apps` option on your
```
Comment (josevalim) [inline lib/phoenix_live_reload/live_reloader.ex:70]:
```suggestion
    changes in sibling applications, set the `reloadable_apps` option on your
```

File: lib/phoenix_live_reload/live_reloader.ex (hunk 1 above)
Comment (nathanl) [reply lib/phoenix_live_reload/live_reloader.ex:87]: sure

**Comment by nathanl** [comment]
@josevalim pushed updates

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**